### ISSUE TITLE
Custom kernel version

### ISSUE THREAD

**Issue by DBLouis** [void-linux/void-mklive#142]
The custom kernel version option `-v linux<version>` is not working as expected. To use the linux-lts package, I had to pass `-p linux4.14` and `-v linux4.14`. Is this expected?

**Comment by q66** [comment]
it does work as expected: https://github.com/void-linux/void-mklive/blob/master/mklive.sh.in#L377 see how it's appended to `PACKAGE_LIST`

i use this feature and it works

**Comment by DBLouis** [comment]
I just tried again:

```
sudo ./mklive.sh -b "base-minimal" -p "linux-lts" -v linux-lts
...
[4/9] Generating initramfs image (xz)...
dracut: Cannot find module directory /lib/modules/4.14_2/
dracut: and --no-kernel was not specified
ERROR: Failed to generate the initramfs
```

What am I doing wrong?

**Comment by q66** [comment]
hm, not quite sure

it shouldn't even let you proceed this far since the script expects a kernel package in format `linux<numerical version>`

**Comment by q66** [comment]
oh, i see why - it's because the check uses `*` so it technically matches as long as there is `linux` in the name

in your case, it doesn't work because if you pass `linux-lts`, which is a metapackage, it'll break, because it doesn't have a specific kernel version (say, `4.14.192`), but only `4.14`, and the script attempts using that to generate the initramfs and it won't work

**Comment by classabbyamp** [comment]
fixed by 68a35230cc2ab226646ba0f07a7d0e08b1bf586e
